core/src/com/goonsquad/galactictd/managers/Assets.java

```diff
@@ -0,0 +1,47 @@
+package com.goonsquad.galactictd.managers;
+
+import com.badlogic.gdx.assets.AssetManager;
+import com.badlogic.gdx.assets.loaders.FileHandleResolver;
+import com.badlogic.gdx.assets.loaders.resolvers.InternalFileHandleResolver;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+import com.badlogic.gdx.graphics.g2d.freetype.FreeTypeFontGenerator;
+import com.badlogic.gdx.graphics.g2d.freetype.FreeTypeFontGeneratorLoader;
+import com.badlogic.gdx.graphics.g2d.freetype.FreetypeFontLoader;
+import com.badlogic.gdx.utils.Disposable;
+
+public class Assets implements Disposable {
+    public AssetManager manager;
+
+    public Assets() {
+        this.manager = new AssetManager();
+    }
+
+    public void loadAssets() {
+        loadLoadingScreenAssets();
+        loadNixieFont();
```

Comment:
Consider the order of calls in this method: do we want to say we are done loading if we have yet to load fonts?